lib/devise/passkeys/controllers/reauthentication_controller_concern.rb

```diff
@@ -58,15 +58,15 @@ def prepare_params
         end
 
         def strategy
-          :passkey_reauthentication
+          :Reauthentication
         end
 
         def auth_options
           { scope: resource_name, recall: root_path }
         end
 
         def delete_reauthentication_challenge
-          session.delete(passkey_reauthentication_challenge_session_key)
+          session.delete(Reauthentication_challenge_session_key)
```

Comment:
This needs to match the casing of the original, since it's the method that's called to get the key for the hash